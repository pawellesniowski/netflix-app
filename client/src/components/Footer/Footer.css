.footer {
    background-color: black;
    color: white;
    padding: 10px 30px;
    position: absolute;
    bottom: 0;
    left: 0;
    right: 0;
    text-align: center;
    font-size: 1.5rem;
}


