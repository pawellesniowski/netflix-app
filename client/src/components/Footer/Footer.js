import React from 'react';
import './Footer.css';

const Footer = () => <footer className="footer">This is footer!</footer>;

export default Footer;
