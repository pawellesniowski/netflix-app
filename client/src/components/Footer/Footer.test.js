import React from "react";
import Footer from "./Footer";
import {shallow} from "enzyme";

describe("Footer Component", () => {
  let component;
  beforeEach(() => {
    component = shallow(<Footer />);
  });

  it("Renders the container", () => {
    expect(component.find('[data-cy="footer"]')).toHaveLength(1);
  });

  it("Matches the snapshot", () => {
    expect(component).toMatchSnapshot();
  });
});
