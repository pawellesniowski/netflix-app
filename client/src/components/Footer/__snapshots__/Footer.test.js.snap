// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Matches the snapshot 1`] = `ShallowWrapper {}`;
